Gaijinizer patch/CommonEvents_180.txt

```diff
@@ -5,8 +5,8 @@
 > CONTEXT: CommonEvents_180/12/Dialogue
 > CONTEXT: CommonEvents_180/19/Dialogue
 > CONTEXT: CommonEvents_180/26/Dialogue
-This devil does not seem to be a partner who
-can persuade...#GT
+This demon doesn't seem to be an opponent
+that can be persuaded...#GTRevisedStrate
 > END STRING
```

Comment:
GT stands for Google Translate, so you can just replace the attribution entirely here to your own.